runs/graphql/server.js

```diff
@@ -29,20 +30,25 @@ const resolvers = {
         const markdown = readFileSync("../../_posts/" + post, "utf8");
         const parsed = fm(markdown);
         const date = parsed.attributes.date
-          ? new Date(parsed.attributes.date + " GMT+0200")
-          : new Date(
-              post
-                .split("-")
-                .slice(0, 2)
-                .join("-") + " GMT+0200"
-            );
+          ? DateTime.fromISO(parsed.attributes.date.toISOString(), { zone: 'Europe/Paris' })
```

Comment:
Why do you convert it back and forth 2 times?

Reply:
@kevinsimper I wanted a DateTime date regardless (to validate) if we infer from post name or if we get a vanilla js date, but I can see how this is confusing